src/lib/logger.ts

```diff
@@ -23,17 +23,16 @@ const additionalProperties = {
 
 export const log = {
     exception: (error: Error | undefined) => {
+        console.error(error);
```

Comment:
Do we want this?

Reply:
Yeah that was intentional, I realized we were only logging in two of the three blocks of the if/else and it was making debugging annoying.